plugins/com.google.cloud.tools.eclipse.util/src/com/google/cloud/tools/eclipse/util/ArtifactRetriever.java

```diff
@@ -146,30 +120,32 @@ public ArtifactRetriever() {
   }
 
   /**
-   * Returns the latest published artifact version, or null if there is no such version.
+   * Returns the latest published non-beta artifact version, or null if there is no such version.
    */
   public ArtifactVersion getLatestArtifactVersion(String groupId, String artifactId) {
     return getLatestIncrementalVersion(idToKey(groupId, artifactId), null);
   }
 
   /**
-   * Returns the latest published artifact version in the version range, or null if there is no such
-   * version.
+   * Returns the latest published non-beta artifact version in the version range, 
+   * or null if there is no such version.
```

Comment:
This can still return a beta artifact, if all versions are beta in the given range.